qiskit_machine_learning/runtime/torch_runtime_client.py

```diff
@@ -0,0 +1,513 @@
+            measurement_error_mitigation: Whether or not to use measurement error mitigation.
+            provider: IBMQ provider that supports runtime services.
+            backend: Selected quantum backend.
+        """
+        # Store settings
+        self._provider = None
+        self._model = None
+        self._loss_func = None
+        self._optimizer = None
+        self._epochs = epochs
+        self._shots = shots
+        self._measurement_error_mitigation = measurement_error_mitigation
+        self._backend = backend
+        self._score_func = None
+
+        # Use setter to check for valid inputs
+        if provider is not None:
+            self.provider = provider
+        self.optimizer = optimizer
+        self.loss_func = loss_func
+        self.model = model
+
+    @property
+    def provider(self) -> Optional[Provider]:
+        """Return the provider."""
+        return self._provider
+
+    @provider.setter
+    def provider(self, provider: Provider) -> None:
+        """Set the provider. Must be a provider that supports the runtime feature."""
+        try:
+            _ = hasattr(provider, "runtime")
+        except QiskitError:
+            # pylint: disable=raise-missing-from
+            raise ValueError(f"The provider {provider} does not provide a runtime environment.")
+
+        self._provider = provider
+
+    @property
+    def model(self) -> Module:
+        """Return the model."""
+        return self._model
+
+    @model.setter
+    def model(self, t_model: Module) -> None:
+        """Set the model."""
+        if not isinstance(t_model, Module):
+            raise TypeError("The model must be an instance of torch.nn.Module")
+        self._model = t_model
+
+    @property
+    def optimizer(self) -> Optimizer:
+        """Return the optimizer."""
+        return self._optimizer
+
+    @optimizer.setter
+    def optimizer(self, optim: Optimizer) -> None:
+        """Set the optimizer."""
+        if not isinstance(optim, Optimizer):
+            raise TypeError("The optimizer must be an instance of torch.optim")
+        self._optimizer = optim
+
+    @property
+    def loss_func(self) -> Union[_Loss, Callable]:
+        """Return the loss function."""
+        return self._loss_func
+
+    @loss_func.setter
+    def loss_func(self, loss: Union[_Loss, Callable]) -> None:
+        """Set the loss function."""
+        if isinstance(loss, _Loss) or callable(loss):
+            self._loss_func = loss
+        else:
+            raise TypeError(
+                "The loss function must be an instance of torch.nn.Loss._Loss or Callable"
+            )
+
+    @property
+    def measurement_error_mitigation(self) -> bool:
+        """Returns whether or not to use measurement error mitigation.
+
+        measurement error mitigation is done using a complete measurement fitter with the
+        ``self.shots`` number of shots and re-calibrations every 30 minutes.
```

Comment:
I don't think there's a `self.shots` property.

Reply:
oops. I forgot to add it. Thx!